client/payment-details/dispute-details/style.scss

```diff
@@ -7,5 +7,49 @@
 
 	.transaction-details-dispute-details-body {
 		padding: $grid-unit-20;
+
+		.dispute-summary-row {
+			margin: 24px 0;
+			.woocommerce-list__item-text {
```

Comment:
While working on #6974, I realise that I'll be making these same changes.

Perhaps these changes should be applied WooPayments-wide to the [`HorizontalList` component CSS](https://github.com/Automattic/woocommerce-payments/blob/9dcc66d2c29e73c1674465d7caf8fc4774009d2b/client/components/horizontal-list/style.scss)?

Reply:
**Update**: the `HorizontalList` list component is **only** used on this screen, so we should change the styles within the component's CSS file.

https://github.com/Automattic/woocommerce-payments/blob/9dcc66d2c29e73c1674465d7caf8fc4774009d2b/client/payment-details/summary/index.tsx#L32

Reply:
Since this is required across 2 PRs and will be a simple change, I'll create a new PR for the style changes to `HorizontalList`.

Reply:
#7126 has been merged, which should allow most of this CSS to be removed 💣